generate_ldscript.py

```diff
@@ -117,8 +117,13 @@ def main(argv):
     else:
         boot_hart = 0
 
+    # Pass sorted memories to the template generator so that the generated linker
+    # script is reproducible.
+    sorted_memories = list(memories.values())
+    sorted_memories.sort(key=lambda m: m["name"])
+
     values = {
-        "memories": list(memories.values()),
+        "memories": sorted_memories,
```

Comment:
What get display for "test ram" in memory block.

Reply:
Basically, if the ram and rom are colocated in the same address range the memory is named "testram".